e2e/test/Common/ConsoleEventListener.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+using System.Linq;
+
+namespace System.Diagnostics.Tracing
+{
+    public sealed class ConsoleEventListener : EventListener
```

Comment:
FYI - this class is referenced in our readme here: https://github.com/Azure/azure-iot-sdk-csharp/tree/master/tools/CaptureLogs#console-logging
Make sure this path is updated as well

Reply:
Thanks for the reference. I'll update it.